src/Model/Photo.php

```diff
@@ -399,109 +399,109 @@ public static function storeByData($uid, $imagedata = "", $url = "")
 			$profiler->saveTimestamp($stamp1, "file", System::callstack());
 		}
 
-		$maximagesize = Config::get('system', 'maximagesize');
+		$maxImageSize = Config::get('system', 'maximagesize');
 
-		if (($maximagesize) && (strlen($imagedata) > $maximagesize)) {
-			$logger->info("Image exceeds size limit.", ['max' => $maximagesize, 'current' => strlen($imagedata)]);
-			return([]);
+		if (($maxImageSize) && (strlen($imagedata) > $maxImageSize)) {
+			$logger->info("image exceeds size limit.", ['max' => $maxImageSize, 'current' => strlen($imagedata)]);
+			return [];
 		}
 
-		$tempfile = tempnam(get_temppath(), "cache");
+		$tempFile = tempnam(get_temppath(), "cache");
 
 		$stamp1 = microtime(true);
-		file_put_contents($tempfile, $imagedata);
+		file_put_contents($tempFile, $imagedata);
 		$profiler->saveTimestamp($stamp1, "file", System::callstack());
 
-		$data = getimagesize($tempfile);
+		$data = getimagesize($tempFile);
 
 		if (!isset($data["mime"])) {
-			unlink($tempfile);
+			unlink($tempFile);
 			$logger->info("File is no picture");
-			return([]);
+			return [];
 		}
 
-		$Image = new Image($imagedata, $data["mime"]);
+		$image = new Image($imagedata, $data["mime"]);
```

Comment:
I like have variable containing object start with a capital.

Reply:
Hm.. What about `$objImage` ? I would prefers not to name any local variable with a started capitalcapital

Reply:
I'm not a fan of the Hungarian notation, and the starting capital for object variables may just a personal preference that doesn't need to persist.